canonical-web-and-design/vanilla-framework
# Align help text and labels for tick elements

## Done

Align help text with tick elements

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3402
Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3966

## QA

- Open https://vanilla-framework-3965.demos.haus/docs/examples/patterns/forms/checkbox
- Open and check help text https://vanilla-framework-3965.demos.haus/docs/examples/patterns/forms/radio
- Check standalon versions also look the same 
https://vanilla-framework-3965.demos.haus/docs/examples/patterns/forms/checkbox
https://vanilla-framework-3965.demos.haus/docs/examples/standalone/patterns/forms/checkbox
https://vanilla-framework-3965.demos.haus/docs/examples/patterns/forms/radio
https://vanilla-framework-3965.demos.haus/docs/examples/standalone/patterns/forms/radio
- Review updated documentation:
  - Added example https://vanilla-framework-3965.demos.haus/docs/base/forms#checkbox

- Also fixed checkbox alignment issue
![fix-checkbox-alignement](https://user-images.githubusercontent.com/14939793/132056962-14f7b863-88ee-4d2e-b362-6dc1c09bebdc.png)


### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
